system/security/shared/seclib.hpp

```diff
@@ -33,6 +34,17 @@
 #define SECLIB "seclib"
 #define LDAPSECLIB "LdapSecurity"
 
+template <typename index_t>
+constexpr uint64_t makeBitMask(index_t bitIndex)
```

Comment:
Can we name these something less likely to conflict like makeSecFeatureMask?

Reply:
They can be renamed, but they are generic functions that just happened to be introduced for use with security features. Would it be better to rename them for this use case or move them to something where a generic name is more appropriate?

Reply:
To be honest I don't want to promote this style of bitmask preparation.  It not as efficient as having the bits pre specified.  It's OK if the bitmasks are established once globally and cached or stored in static variables.  I would usually prefer having the bits set in the enumeration and used directly.

```
enum SecManagerFeatureIndex : unsigned
{
    SMF_CreateUser = 0x000001,
    SMF_CreateResourceList = 0x000002,
    SMF_Subscribe = 0x000004,

```
etc.

That would be more efficient for both the plugin and the user.  Functions for setting AllExcept, etc could still be used.